UITests/FAQ/FaqUITests.swift

```diff
@@ -41,3 +40,18 @@ class FaqUITests: AppViewTestCase, ViewTestCase {
     ]
   }
 }
+
+extension FAQ {
+  static var mockedFAQ: FAQ {
+    FAQ(
+      title: "Immuni dice che potrei essere a rischio, ma io mi sento bene. Cosa devo fare?",
+      content: """
+       Ti suggeriamo vivamente di seguire tutte le raccomandazioni di Immuni. Ci sono molte persone asintomatiche che hanno diffuso il virus senza rendersene conto. Uno dei punti di forza di Immuni è proprio la capacità di avvertire queste persone. Per favore, fai la tua parte seguendo le raccomandazioni, anche se pensi di non essere contagioso.
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: UITests/FAQ/FaqUITests.swift;
  Line: 49;
-->

- :warning: swiftlint (0.39.2): Line should be 130 characters or less: currently 347 characters (line_length)